lib/errors.js

```diff
@@ -2,9 +2,10 @@
 
 const util = require('util');
 
-function error(code, data, err) {
-    if (util.isError(code))
-        return code;
+function error(code, data, err, stackTrace) {
+    if (util.isError(code)) {
+        return error(500, null, code);
```

Comment:
why would we call an error function to return only the error? I tmakes no sense, otherwise just use the error